yast/README.md

```diff
@@ -0,0 +1,89 @@
+# Shim for YaST
```

Comment:
do you think we should keep this readme? If so, it should be updated to include why the log and verify callbacks are needed. And the Testing steps too.

Reply:
I don't know really. We don't have anything better right now and this is pretty low prio considering that there are no new clients for this library in sight.